pkgs/by-name/op/openfortivpn-webview/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, electron
+}:
+
+let
+  version = "1.2.0";
+in
+
+buildNpmPackage {
+  inherit version;
+
+  pname = "openfortivpn-webview";
```

Comment:
```suggestion
in
buildNpmPackage {
  pname = "openfortivpn-webview";
  inherit version;
```

Reply:
thanks for the suggestion, updated that accordingly! also noticed that RFC-166 formatting came through in the meanwhile, so got that reformatted too (after two retries).